_posts/2025-01-15-moving-data-between-JBOD-disks-using-cruise-control.md

```diff
@@ -0,0 +1,546 @@
+          "logDir": "/var/lib/kafka/data-0/kafka-log4"
+        },
+        {
+          "partitions": [
+            {
+              "partition": "my-topic-2",
+              "size": 0,
+              "offsetLag": 0,
+              "isFuture": false
+            }
+          ],
+          "error": null,
+          "logDir": "/var/lib/kafka/data-1/kafka-log4"
+        }
+      ]
+    },
+    {
+      "broker": 5,
+      "logDirs": [
+        {
+          "partitions": [
+            {
+              "partition": "my-topic-2",
+              "size": 0,
+              "offsetLag": 0,
+              "isFuture": false
+            }
+          ],
+          "error": null,
+          "logDir": "/var/lib/kafka/data-1/kafka-log5"
+        },
+        {
+          "partitions": [
+            {
+              "partition": "my-topic-0",
+              "size": 0,
+              "offsetLag": 0,
+              "isFuture": false
+            }
+          ],
+          "error": null,
+          "logDir": "/var/lib/kafka/data-0/kafka-log5"
+        },
+        {
+          "partitions": [
+            {
+              "partition": "my-topic-1",
+              "size": 0,
+              "offsetLag": 0,
+              "isFuture": false
+            }
+          ],
+          "error": null,
+          "logDir": "/var/lib/kafka/data-2/kafka-log5"
+        }
+      ]
+    },
+    {
+      "broker": 3,
+      "logDirs": [
+        {
+          "partitions": [
+            {
+              "partition": "my-topic-0",
+              "size": 0,
+              "offsetLag": 0,
+              "isFuture": false
+            }
+          ],
+          "error": null,
+          "logDir": "/var/lib/kafka/data-0/kafka-log3"
+        },
+        {
+          "partitions": [
+            {
+              "partition": "my-topic-2",
+              "size": 0,
+              "offsetLag": 0,
+              "isFuture": false
+            }
+          ],
+          "error": null,
+          "logDir": "/var/lib/kafka/data-2/kafka-log3"
+        },
+        {
+          "partitions": [
+            {
+              "partition": "my-topic-1",
+              "size": 0,
+              "offsetLag": 0,
+              "isFuture": false
+            }
+          ],
+          "error": null,
+          "logDir": "/var/lib/kafka/data-1/kafka-log3"
+        }
+      ]
+    }
+  ],
+  "version": 1
+}
+```
+
+Next, let's move the data from volumes 1 and 2 to volume 0 for all the brokers. For doing that let's create a `KafkaRebalance` resource with `remove-disks` mode.
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaRebalance
+metadata:
+  name: my-rebalance
+  labels:
+    strimzi.io/cluster: my-cluster
+# no goals specified, using the default goals from the Cruise Control configuration
+spec:
+  mode: remove-disks
+  moveReplicasOffVolumes:
+    - brokerId: 3
+      volumeIds: [1, 2]
+    - brokerId: 4
+      volumeIds: [1, 2]
+    - brokerId: 5
+      volumeIds: [1, 2]
```

Comment:
MAybe you can use the same YAML formatting as you have below in the output from Kubernetes? It might help to avoid confusion between the meaning of
```
volumeIds: [1, 2]
```
and 
```
      volumeIds:
        - 1
        - 2
```
among less experienced users.